packages/eslint-plugin-turbo/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../../docs/public/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "dependsOn": ["build"]
```

Comment:
Same, there's no build task in the pipeline.

Reply:
same comment as previous! it should work